edifact-generator/inbound-generator.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import os
+from templates import Inbound
+
+parser = argparse.ArgumentParser(prog=__file__, description='Generate inbound EDIFACT interchanges as .dat files')
+parser.add_argument('--count', type=int, help='the number of files to generate - minimum 1 maximum 9999',
+                    default='1', required=False)
+
+args = parser.parse_args()
+
+
+def run():
+    validate_count_argument()
+    create_output_folder()
+    clear_output_folder()
```

Comment:
why the folder, that has just been created, is then cleared?

Reply:
maybe there should be a condition: if folder exists, then clear, if don't exist, then create

Reply:
For continuous usage - if it was used before it should be cleared. For situation like: previously I created 500 files and now I want only 5

Reply:
If it was just created (first time use) then folder is empty and clearing will just do nothing

Reply:
It will do nothing, but the unnecessary function is called. But I am ok with leaving it like this if everybody agree. It is not very time-consuming, just little inconvenience while reading the code.

Reply:
My reasoning for that is that `output` folder will be empty just 1 time (first use) then it should always have files in it so checking that would be pointless after first run.